2021/d01/p2.py

```diff
@@ -0,0 +1,34 @@
+from typing import Iterator
+
+from p1 import INPUT_FILE_PATH, get_measurements_from_input, get_depth_measurements_increases
+
+_DEFAULT_WINDOW_SIZE = 3
+
+
+def get_measurements_windows(measurements: Iterator[int], window_size: int = _DEFAULT_WINDOW_SIZE) -> list[tuple[int]]:
```

Comment:
Think how you can use zip for the general window size. Hint: argument list unpacking.

Reply:
Was 5675ce92 what you expected?